content/automate/ManageIQ/PhysicalInfrastructure/CiscoIntersight/Services/DeployServerProfileTemplate.class/__methods__/deploy_server_profile_template.rb

```diff
@@ -1,8 +1,13 @@
-template_id = $evm.root.attributes['dialog_template']
 profile_name = $evm.root.attributes['dialog_name']
 server_id = $evm.root.attributes['dialog_server']
+service_template_object = $evm.root['service_template_provision_task'].source
+template_id = service_template_object.options[:server_profile_template_id]
+
+if template_id.nil? || (template_id == "")
```

Comment:
You should be able to use `.blank?` here...
```suggestion
if template_id.blank?
```